src/components/Modal/GPOSWizard/GPOSWizard.scss

```diff
@@ -0,0 +1,316 @@
+@import '../../../assets/stylesheets/mixins';
+@import '../../../assets/stylesheets/variables';
+
+$breakpoints: (
+  default: 1920px,
+  mac: 1531px,
+  sm: 1275px
+);
+
+//
+// RESPOND BELOW
+//––––––––––––––––––––––––––––––––––––––––––––––––––
+// @include respond-to(mac) {}
+@mixin respond-to($breakpoint) {
+  // If the breakpoint exists in the map.
+  @if map-has-key($breakpoints, $breakpoint) {
+    // Get the breakpoint value.
+    $breakpoint-value: map-get($breakpoints, $breakpoint);
+    // Write the media query.
+    @media (max-width: ($breakpoint-value - 1)) {
+      @content;
+    }
+  }
+  // If the breakpoint doesn't exist in the map.
+  @else {
+
+    // Log a warning.
+    @warn 'Invalid breakpoint: #{$breakpoint}.';
+  }
+}
+
+@mixin gpos-start-btn {
+  @include respond-to(mac) {
+    width: 133px;
+    height: 58px;
+  }
+
+  width: 160px;
+  height: 70px;
+  border-radius: 10px;
+}
+
+.gpos-modal {
+  @include respond-to(mac) {
+    max-width: 1255px;
+    max-height: 726px;
+  }
+
+  @include respond-to(sm) {
+    max-width: 80%;
+    border-radius: 10px 10px 0 0;
+  }
+
+  position: relative;
+  width: 100%;
+  max-width: 1505px;
+  height: 100%;
+  max-height: 869px;
+  margin: 0 auto;
+  border-radius: 15px 0 0 15px;
+  background: transparent radial-gradient(closest-side at 50% 52%, rgb(61, 93, 203) 0%, rgb(49, 53, 121) 100%);
+
+  &__bg {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: 2;
+    margin: 0 auto;
+    background: rgba(0, 0, 0, 0.5);
+    overflow: auto;
+    resize: both;
+  }
+
+  &__btns {
+    @include respond-to(mac) {
+      width: 485px;
+      margin: 48px auto;
+    }
+
+    display: flex;
+    justify-content: space-between;
+    width: 583px;
+    margin: 62px auto;
+  }
+
+  &__btn {
+    @include gpos-start-btn();
+
+    &-cancel, &-up, &-down {
+      @extend .gpos-modal__btn;
+      @include transition(all .2s ease);
+
+      border: 2px solid transparent;
+
+      &:active {
+        border-color: rgb(51, 183, 239);
+      }
+    }
+
+    &-cancel {
+      background: rgb(41, 58, 143);
+
+      &:hover {
+        background: lighten(rgb(41, 58, 143), $amount: 10%);
+      }
+
+      &:active {
+        background: darken(rgb(41, 58, 143), $amount: 10%);
+        border-color: $color-add2-bd-color;
+      }
+    }
+
+    &-up {
+      background: rgb(51, 183, 239);
+
+      &:hover {
+        background: lighten(rgb(51, 183, 239), $amount: 10%);
+      }
+
+      &:active {
+        background: darken(rgb(51, 183, 239), $amount: 10%);
+      }
+    }
+
+    &-down {
+      @include respond-to(mac) {
+        width: 172px;
+      }
+
+      width: 207px;
+      margin-right: 0;
+      background: rgb(19, 212, 183);
```

Comment:
not going to comment every line. But just look at all the rgb references and throw them in a color constants file